src/test/resources/features/profile.feature

```diff
@@ -0,0 +1,46 @@
+Feature: Profile
+
+  Background:
+    Given that I enter the conduit site
+    And I click on the Sign Up link
+    And I register as a new user
```

Comment:
You can also edit the profile after the sign in. I think this is ok, but why did you choose this flow over the sign in one?

Reply:
So as not to be modifying the same user and to be able to see the functionality with different users